get.js

```diff
@@ -3,10 +3,21 @@ var document = require("global/document");
 module.exports = getJSONGlobal;
 
 var globals;
-function getJSONGlobal(key) {
+
+/**
+ * @function getJSONGlobal
+ * @param {String} key key on globals to get
+ * @param {Object} [opts] options object
+ * @param {Function} [opts.parse = JSON.parse] parse function to run on the textContent, defaults to JSON.parse.
+ *                                             available since v2.1.0
+ */
+function getJSONGlobal(key, opts) {
+  opts = opts || {};
+  opts.parse = opts.parse || JSON.parse;
+
     if (!globals) {
       var jsonGlobalsElement = document.getElementById('json-globals');
-      globals = JSON.parse(jsonGlobalsElement.textContent);
+      globals = opts.parse(jsonGlobalsElement.textContent);
```

Comment:
Does this introduce a security degradation? Let's say you pass "eval" and serialize-javascript is somewhow compromised, or the page is misrendered somehow due to an unrelated XSS? Or maybe I'm paranoid?

Reply:
if someone can hijack the `json-globals` element on the page before `eval` is called, then yes it would be XSS vuln